pub/index.html

```diff
@@ -1,37 +1,155 @@
+                font-size: 40px;
+                color: #333333;
+                font-family: 'Open Sans', sans-serif;
+            }
+            
+            
+            h3 {
+                font-size: 40px;
+                color: #333333;
+                font-family: 'Open Sans', sans-serif;
+            }
+            
+            p {
+                float: left;
+                text-align: left;
+                padding-right: 10%;
+                width: 75%;
+                line-height: 26px;
+                font-family: 'PT Serif', serif;
+                font-size: 17px;    
+                color: #666666;
+            }
+        
+           
+            
+            aside h1 {
+                font-size: 16px;
+                line-height: 19px;
+                font-family: 'Open Sans', sans-serif;
+                color: #666;
+            }
+            
+            aside h2 {
+                font-size: 20px;
+                padding-bottom: 1px;
+                margin-bottom: 1px;
+            }
+           
+           .blog-updates {
+               font-size: 17px;
+               line-height: 26px;
+               font-family: 'PT Serif', serif;
+               text-align: left;
+               padding-right: 60px;
+           }
+            
+         .page-column-side {
+              float: left;
+              width: 25%;
+              min-width: 160px;
+              text-align: left;
+              padding-right: 60px;
+              word-wrap: break-word;
+              position: relative;
+              margin-top: -35px;
+              margin-left:50px;
+            }
+
+            .page-column-main {
+              display:table;
+              text-align: left;
+
+
+            }
+            
+            footer aside {
+              float: left;
+              width: 30%;
+                
+            }
+
+            .content {
+              width: 70%;
+              
+            }
+
+            footer aside img {
+              float: right;
+              margin: 30px;
+              border-radius: 50%;
+              overflow: hidden;
+            }
+            
+            
+        </style>
+        <script src="js/vendor/modernizr-2.8.3.min.js"></script>
+    </head>
+    <body>
+        <!--[if lt IE 8]>
+            <p class="browserupgrade">You are using an <strong>outdated</strong> browser. Please <a href="http://browsehappy.com/">upgrade your browser</a> to improve your experience.</p>
+        <![endif]-->
+
+        <!-- Add your site or application content here -->
+        <div class="container" id="container">
+            <header id="page-header">
+                <img src="http://static.tumblr.com/3c7cd96cde56779343d2e9a22afec085/w5gxxfv/oCrn8vkw8/tumblr_static_9cxjqqnfamsccsg4ow84sg8kg.jpg"/>
+            </header> <!-- #page-header -->
+            <section class="page-columns">
+               <aside style="float:left; width:100px" class="page-column-side">
```

Comment:
Best to refactor these inline styles out of your HTML, as they can be very confusing to debug.